src/pages/jeongeun/Main/Feeds.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react';
+import CommentBox from './CommentBox';
+
+const Feeds = () => {
+  const [feeds, setFeeds] = useState([]);
+
+  useEffect(() => {
+    fetch('data/feedData.json')
+      .then((result) => result.json())
+      .then((feeds) => setFeeds(feeds));
+  }, []);
+
+  return (
+    <div className="feedsBox">
+      {feeds.map((feeds) => (
```

Comment:
feeds 배열에서 각각의 요소를 나타내는 매개변수이기 때문에, feeds -> feed가 적절해보임
```suggestion
      {feeds.map((feed) => (
```